packages/notebook-extension/schema/tracker.json

```diff
@@ -559,21 +559,13 @@
     {
       "command": "notebook:run-cell",
       "keys": ["Ctrl Enter"],
+      "macKeys": ["Ctrl Enter", "Cmd Enter"],
       "selector": ".jp-Notebook:focus"
     },
     {
       "command": "notebook:run-cell",
       "keys": ["Ctrl Enter"],
-      "selector": ".jp-Notebook.jp-mod-editMode"
-    },
-    {
-      "command": "notebook:run-cell",
-      "keys": ["Accel Enter"],
-      "selector": ".jp-Notebook:focus"
-    },
-    {
-      "command": "notebook:run-cell",
-      "keys": ["Accel Enter"],
+      "macKeys": ["Ctrl Enter", "Cmd Enter"],
```

Comment:
This means that you need to press the chord `Ctrl Enter, Cmd Enter`, i.e., you need to press both key combinations in succession for the shortcut to trigger. To define two separate shortcuts, you need to make two completely separate entries.

Reply:
I would suggest reverting this part of the PR - it was very intentional that there were two shortcuts, one for ctrl enter and one for Accel enter. If you wanted to clean things up, you could make the Ctrl-enter shortcut only a mackeys one, but I don't think it is necessary either.

Reply:
Thanks @jasongrout I did not understand the list thingy properly. But then the shortcuts setting loader does not either :wink: as seen in https://github.com/jupyterlab/jupyterlab/issues/12103 defining multiple shortcuts for the same command is not possible.

I will look at correcting the loader.

Reply:
This works only partially, we still have one warning (instead of two):

```
{ command: "notebook:run-cell", keys: (1) […], selector: ".jp-Notebook:focus" }
```

That comes from `Cmd Enter` being converted as `Enter` that conflict with another existing shortcuts. Actually it seems wrong to even accept that shortcut on non Mac platform as it can lead to unexpected behavior.

Reply:
Can you put the `Cmd Enter` in the macKeys argument?

Reply:
It looks like `Cmd` is just ignored as a modifier on non-mac platforms. I agree that this seems problematic - the shortcut should probably be rejected if it has Cmd on a non-mac:

https://github.com/jupyterlab/lumino/blob/aae9de3a042b91bd54ba7f80b2ff7034a879fe4e/packages/commands/src/index.ts#L1146-L1148

Reply:
Or you can leave the Accel Enter shortcut, and make a new one that is `Ctrl Enter` in the macKeys, so it just applies for macs.

Reply:
> Can you put the Cmd Enter in the macKeys argument?

I'm not sure to get that suggestion. `keys` cannot be [undefined](https://github.com/jupyterlab/jupyterlab/blob/aef2a162ece7f2e983bb118675669920c6bed6f9/packages/shortcuts-extension/schema/shortcuts.json#L40). And if I set the command with:

```
"keys": ["Ctrl Enter"],
"macKeys": ["Cmd Enter"]
```

It will override the "Ctrl Enter" shortcut as [lumino](https://github.com/jupyterlab/lumino/blob/aae9de3a042b91bd54ba7f80b2ff7034a879fe4e/packages/commands/src/index.ts#L1164) is taking `macKeys || keys`. But we want to have both "Ctrl Enter" and "Cmd Enter" on MacOS.

Reply:
keys can be an empty list, though.